examples/rhino-sdk/streamlit-visualization/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas
+matplotlib
```

Comment:
We probably should have specific versions for this because pandas and matplotlib do in fact have breaking changes